feat(services): add professional services section (#48)
AshAvalanche/ash-docs: docs/professional-services/index.md

---
sidebar_position: 1
sidebar_label: Overview
---

> # Professional Services Overview

Reviewer: ```suggestion
# Professional Services
```

See examples of the team contributions:

- [Ava Labs repositories](https://github.com/issues?q=is%3Aissue+involves%3ANuttymoon+involves%3Aleopaul36+archived%3Afalse+org%3Aava-labs+)

#### Node operating

Ash is **operating multiple Avalanche nodes** (on the mainnet and Fuji testnet), and taking part in decentralized RPC networks such as [Lava](https://lavanet.xyz).

### Education

[Gauthier](https://www.linkedin.com/in/gauthier-leonard/) has been a **distributed systems teacher** at some of the top French engineering schools ([CentraleSupélec](https://www.centralesupelec.fr/), [ECE Paris](https://www.ece.fr/), [DSTI](https://www.datasciencetech.institute/fr/)) for more than 3 years.

## How we can help you

Our team can offer you the following services:

### Avalanche nodes setup and maintenance

Ash [open-source tools](/docs/toolkit/) and premium [Console](/docs/console/) plans can significantly reduce the setup and maintenance cost of your Avalanche nodes.

Our team can help you with the **initial setup** and provide you with a maintenance plan **tailored to your needs**.

> ### Extra features for the Ash tools

Reviewer: ```suggestion
### Custom features for the Ash tools
```

> Because we are the maintainers of the Ash [open-source tools](/docs/toolkit/) and [Console](/docs/console/), we can **develop extra features** for any extra need you may have. This could be:

Reviewer: ```suggestion
As the maintainers of the Ash [open-source tools](/docs/toolkit/) and [Console](/docs/console/), we can **develop custom features** for any additional need you may have. This could be:
```

Author: For me "extra" is more accurate as the features would likely be open-sourced as part of the toolkit.

Reviewer: How about "additional" then? Extra make it sound like they **won't** be open-sourced IMO.

- Better support for your cloud provider
- Support for a new blockchain runtime (VM)
- Integration with your monitoring system
- Etc.

### Subnet (AppChain) design and development

Avalanche [Subnets](https://docs.avax.network/learn/avalanche/subnets-overview) allow any project to create **its dedicated blockchain** (or "AppChain") with their own set of rules and validators. Subnets can run **custom software** while benefiting from the **Avalanche consensus layer**.

Our team can help you **take the best decisions** regarding your Subnet design and follow you through the development process, **from devnet to mainnet**.

### Private blockchain network design and setup

> The flexibility of Avalanche makes it a great candidate to set up **private blockchain networks** with your partners. Each use case can be run on its own Subnet, with its own set of rules and runtime.

Reviewer: ```suggestion
The flexibility of Avalanche makes it the perfect tool to set up **private blockchain networks** with your partners. Each use case can be run on its own Subnet, with its own set of rules and runtime.
```
